app/views/layouts/document_upload.html.erb

```diff
@@ -40,8 +40,12 @@
           <%# document upload form %>
           <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, id: "file-upload-form", class: "spacing-below-60" do |f| %>
             <div class="document-upload">
-              <div class="file-upload">
-                <%= f.file_field(:document, class: "form__documentuploader file-input", data: { "upload-immediately" => true}) %>
+              <div class="file-upload form-group <%= f.send(:error_state, @form, :document) %>">
+                <%= f.file_field(
+                  :document,
+                  class: "form__documentuploader file-input#{f.send(:error_state, @form, :document)}",
+                  data: { "upload-immediately" => true}
```

Comment:
Unfortunately since this is not using a form builder public helper method I am calling a private method to get the error styling into the inputs. Maybe investing some time to create an enhanced cfa_file_field in VitaMinFormBuilder would help clean this up?